README.md

```diff
@@ -18,8 +18,8 @@ Add to `Move.toml`:
 
 ```toml
 [dependencies.UQ64x64]
-git = "https://github.com/pontem-network/UQ64x64.git"
-rev = "v0.3.6"
+git = "https://github.com/ThalaLabs/UQ64x64.git"
+rev = "v0.3.7"
```

Comment:
we are not releasing as pontem does https://github.com/pontem-network/uq64x64/releases/tag/v0.3.6
so probably just use commit hash here

Reply:
agreed. Maintaining releases seems to be too much overhead